src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -61,6 +61,8 @@ See xref:zeromq.adoc[ZeroMQ Support] for more information.
 Instead of throwing `IllegalStateException`, the `RedisLockRegistry.RedisLock.unlock()` method throws `ConcurrentModificationException` if the ownership of the lock is expired.
 See xref:redis.adoc[Redis Support] for more information.
 
+A method `RedisLockRegistry.setRenewalTaskScheduler` is added to configure the scheduler for automatic renewal of locks. When it is set, the lock will be automatically renewed every 1/3 of the expiration time after the lock is successfully acquired, until unlocked or the redis key is removed
```

Comment:
No need to go in details in the `whats-new`.
The link above would lead us to the target doc.
So, logically this sentence has to be moved before that `See xref:redis.adoc[Redis Support] for more information.`